From 6afe13848dc88dcd895db4e6f29e5b3fd663dbc2 Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Tue, 9 Oct 2012 09:05:19 +0200
Subject: [PATCH][BUILD] Build fix.

Add missing autogen.sh for the buld API and AM_PROG_AR to configure.ac.

Signed-off-by: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
---
 autogen.sh   | 22 ++++++++++++++++++++++
 configure.ac |  2 ++
 2 files changed, 24 insertions(+)
 create mode 100755 autogen.sh

diff --git a/autogen.sh b/autogen.sh
new file mode 100755
index 0000000..c04283c
--- /dev/null
+++ b/autogen.sh
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+test -n "$srcdir" || srcdir=`dirname "$0"`
+test -n "$srcdir" || srcdir=.
+
+olddir=`pwd`
+cd $srcdir
+
+[ ! -e ChangeLog ] && touch ChangeLog
+
+AUTORECONF=`which autoreconf`
+if test -z $AUTORECONF; then
+        echo "*** No autoreconf found, please intall it ***"
+        exit 1
+fi
+
+mkdir -p m4
+
+autoreconf --force --install --verbose
+
+cd $olddir
+test -n "$NOCONFIGURE" || "$srcdir/configure" "$@"
diff --git a/configure.ac b/configure.ac
index 69eb13b..a5ab855 100644
--- a/configure.ac
+++ b/configure.ac
@@ -24,6 +24,8 @@ AC_CONFIG_HEADERS(config.h)
 AM_INIT_AUTOMAKE([1.10 -Wall -Werror -Wno-override])
 AM_SILENT_RULES([yes])
 
+AM_PROG_AR
+
 # Library code modified:                              REVISION++
 # Interfaces changed/added/removed:   CURRENT++       REVISION=0
 # Interfaces added:                             AGE++
-- 
1.7.12.2

